tensorflow/.buildspec_templates/std_buildspec_template.yml

```diff
@@ -0,0 +1,94 @@
+
+  images:
+    BuildTensorflowCPUTrainPy37DockerImage:
+      <<: *TRAINING_REPOSITORY
+      build: &TENSORFLOW_CPU_TRAINING_PY3 false
+      image_size_baseline: 1000
+      device_type: &DEVICE_TYPE cpu
+      python_version: &DOCKER_PYTHON_VERSION py3
+      tag_python_version: &TAG_PYTHON_VERSION py37
+      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION ]
+      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+      context:
+        <<: *TRAINING_CONTEXT
+    BuildTensorflowGPUTrainCu102Py37DockerImage:
+      <<: *TRAINING_REPOSITORY
+      build: &TENSORFLOW_GPU_TRAINING_PY3 false
+      image_size_baseline: 1000
+      device_type: &DEVICE_TYPE gpu
+      python_version: &DOCKER_PYTHON_VERSION py3
+      tag_python_version: &TAG_PYTHON_VERSION py37
+      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION ]
+      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+      context:
+        <<: *TRAINING_CONTEXT
+    BuildTensorflowExampleGPUTrainCu102Py37DockerImage:
+      <<: *TRAINING_REPOSITORY
+      build: &TENSORFLOW_GPU_TRAINING_PY3 false
+      image_size_baseline: 1000
+      base_image_name: BuildTensorflowGPUTrainCu110Py37DockerImage
+      device_type: &DEVICE_TYPE gpu
+      python_version: &DOCKER_PYTHON_VERSION py3
+      tag_python_version: &TAG_PYTHON_VERSION py37
+      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION,
+                   "-example" ]
+      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /example, /Dockerfile., *DEVICE_TYPE ]
+      context:
+        <<: *TRAINING_CONTEXT
+    BuildTensorflowCPUInferencePy37DockerImage:
+      <<: *INFERENCE_REPOSITORY
+      build: &TENSORFLOW_CPU_INFERENCE_PY3 false
+      image_size_baseline: 1000
+      device_type: &DEVICE_TYPE cpu
+      python_version: &DOCKER_PYTHON_VERSION py3
+      tag_python_version: &TAG_PYTHON_VERSION py37
+      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *OS_VERSION ]
+      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /Dockerfile., *DEVICE_TYPE ]
+      context:
+        <<: *INFERENCE_CONTEXT
+    BuildTensorflowGPUInferenceCu102Py37DockerImage:
+      <<: *INFERENCE_REPOSITORY
+      build: &TENSORFLOW_GPU_INFERENCE_PY3 false
+      image_size_baseline: 1000
+      device_type: &DEVICE_TYPE gpu
+      python_version: &DOCKER_PYTHON_VERSION py3
+      tag_python_version: &TAG_PYTHON_VERSION py37
+      tag: !join [ *VERSION, "-", *DEVICE_TYPE, "-", *TAG_PYTHON_VERSION, "-", *CUDA_VERSION, "-", *OS_VERSION ]
+      docker_file: !join [ docker/, *SHORT_VERSION, /, *DOCKER_PYTHON_VERSION, /, *CUDA_VERSION, /Dockerfile., *DEVICE_TYPE ]
+      context:
+        <<: *INFERENCE_CONTEXT
```

Comment:
isn't there example image for Inference?

Reply:
No, Example images only exist for GPU training DLC images.